### PR TITLE
More common terms

### PR DESCRIPTION
**Pull request by fenomas**
Neat idea! Adding a few terms I hear from programmers. Let me know if you have comments.

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -144,6 +152,7 @@ Beta version | ベータ版 | ベータばん |
 Closed beta | クローズドベータ | |
 Front-end | フロントエンド | |
 Management | 管理 | かんり | Of people, software, etc
+Infrastructure | インフラ |
```
Comment (ronkorving) [inline ReadMe.md]: Please add one last pipe symbol (like you see a few lines above this one).

File: ReadMe.md [hunk 2]
```diff
@@ -106,6 +106,14 @@ To select | 選択（する） | せんたく |
 To search | 検索（する） | けんさく |
 Object Oriented Programming (OOP) | オブジェクト指向 | オブジェクトしこう |
 Functional Programming (FP) | 関数型プログラミング | かんすうがたプログラミング |
+Inherit (from X) | （Xを）継承（する） | けいしょう |
+Type | 型 | かた | as in data types
+Static | 静的 | せいてき |
+Dynamic | 動的 | どうてき |
+Optimize | 最適化（する） | さいてきか |
```
Comment (ronkorving) [inline ReadMe.md]: "To optimize" please (English verbs prefixed with "to").

Comment (ronkorving) [inline ReadMe.md]: "To optimize" please (English verbs prefixed with "to").

File: ReadMe.md [hunk 3]
```diff
@@ -106,6 +106,14 @@ To select | 選択（する） | せんたく |
...
+Regexp | 正規表現 | せいきひょうげん |
```
Comment (ronkorving) [inline ReadMe.md]: "Regular expression" please :)

**Comment by fenomas** [comment]
Fixes made! There are several other verbs that could be fixed (development, execution, etc.) but I'll leave that to you.

**Comment by ronkorving** [comment]
Feedback on any mistakes you see is very welcome! (or are they stylistic issues?)

I'm aware some words have not been perfectly translated, and we could use help on polishing that (I want at least every translation double-confirmed). Feel free to open an issue with your thoughts.

File: ReadMe.md [hunk 4]
```diff
@@ -106,6 +106,14 @@ To select | 選択（する） | せんたく |
 To search | 検索（する） | けんさく |
 Object Oriented Programming (OOP) | オブジェクト指向 | オブジェクトしこう |
 Functional Programming (FP) | 関数型プログラミング | かんすうがたプログラミング |
+Inherit (from X) | （Xを）継承（する） | けいしょう |
+Type | 型 | かた | as in data types
+Static | 静的 | せいてき |
+Dynamic | 動的 | どうてき |
```
Comment (ronkorving) [inline ReadMe.md:112]: Are these "static" and "dynamic" in the broadest sense of the word? Or do they have a very specific usage domain?

Comment (fenomas) [reply ReadMe.md:112]: Um. Both? They are general terms but also used specifically for e.g. static/dynamic types, static code analysis, etc.

**Comment by ronkorving** [comment]
Had it reviewed by a Japanese coworker. Good to go, thanks!

**Pull request merged**